helm/falco-app/charts/falco-exporter/templates/daemonset.yaml

```diff
@@ -28,7 +28,7 @@ spec:
         - name: {{ .Chart.Name }}
           securityContext:
             {{- toYaml .Values.securityContext | nindent 12 }}
-          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          image: "{{ .Values.image.registry }}/{{ .Values.image.repository }}:{{ .Values.image.tag }}"
```

Comment:
We should not make these changes in the subchart. It will break when we update from upstream. It would be nicer to have everything in the most top level `values.yaml` - just to make maintenance down the road easier.

Reply:
Changed this upstream